src/main/webapp/quiz/question4.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<head>
+	<title>
+		Need Help?
+	</title>
+	<link rel="stylesheet" href="quiz.css">
+	<div class="head-style">
+		FUTURE STYLE
+	</div>
+</head>
+
+<body>
+	</br></br>
+	<div class="border">
+		<div class="header">
+			<b>What Is Your Emergancy?</b>
+		</div>
+		
+		<div class="question">
+			<b>Do you know what drug the victim took?</b>
+			
+			<div class="Note">
+				<i>Note: A person is unconscious when they do not respond to verbal or painful stimulus. A response can be 
+				verbal acknowledgement from the victim, a grunt or wince to pain, or their eyes following you.</i>
+			</div>
+		</div>
+		
+		<div class="answer-left">
+			<a href="question5.html">Alcohol</a>
+		</div>
+		
+		<div class="answer-right">
+			<a href="question4.html">No, the victim is conscious</a>

```

Comment:
This is a self-link. I think from the picture that this should allow choosing from a list of drugs?